packages/ui/jest-setup.ts

```diff
@@ -1,8 +1,20 @@
 import '@testing-library/jest-dom';
+import { FilterDOMPropsKeys, filterDOMProps } from 'uniforms';
 // import '@testing-library/jest-dom/extend-expect'
 
-beforeAll(() => {
-  jest
-    .spyOn(global, 'crypto', 'get')
-    .mockImplementation(() => ({ getRandomValues: () => [12345678] }) as unknown as Crypto);
+filterDOMProps.register('inputRef' as FilterDOMPropsKeys, 'placeholder' as FilterDOMPropsKeys);
+
+jest
+  .spyOn(global, 'crypto', 'get')
+  .mockImplementation(() => ({ getRandomValues: () => [12345678] }) as unknown as Crypto);
+
+jest.spyOn(console, 'warn').mockImplementation((...args) => {
```

Comment:
Just a question - this is for "restricting" the logging when testing?

Reply:
This is to ignore the known logs, for instance, there's an open issue in [@patternfly/react-topology](https://github.com/patternfly/react-topology/issues/70) that logs a warning when importing some tokens.

We're doing this to cleanup the log from the test stage.

Reply:
thanks!